app/uk/gov/hmrc/gform/models/FieldValueRaw.scala

```diff
@@ -18,13 +18,30 @@ package uk.gov.hmrc.gform.models
 
 import cats.data.NonEmptyList
 import cats.syntax.all._
-import play.api.libs.json._
+import play.api.libs.json.{ OFormat, _ }
 import uk.gov.hmrc.gform.core.Opt
 import uk.gov.hmrc.gform.exceptions.{ InvalidState, UnexpectedState }
 import FieldValueRaw._
+import play.api.libs.functional.syntax._
 
 object FieldValueRaw {
-  implicit val format: OFormat[FieldValueRaw] = Json.format[FieldValueRaw]
```

Comment:
not possible because of the recursiveness of FieldValueRaw. will discuss more when I'm in